Added examples section to application.yml

andrewjohnysons opened the pull request:
Added data to application.yml
Written config class to load the data and create maps
RestController uses maps to fulfill data responses.

andrewjohnysons committed:
Added data to application.yml
Written config class to load the data and create maps
RestController uses maps to fulfill data responses.

andrewjohnysons committed:
Added data to application.yml
Written config class to load the data and create maps
RestController uses maps to fulfill data responses.
Added new Fields to DTOs and deleted local versions

andrewjohnysons committed:
Merge branches 'CR-463' and 'master' of github.com:ONSdigital/census-mock-case-api-service into CR-463

cookel2 reviewed (commented):
I have pulled down the changes into my local and tested to see if I can access the new data. I have accessed the case details successfully, when running the fake case service with RabbitMQ and the Contact Centre service, from the following two URLs: http://localhost:8171/cases/3305e937-6fb1-4ce1-9d4c-077f147789ac and http://localhost:8171/cases/3305e937-6fb1-4ce1-9d4c-077f147789ab. However, I still need to test the qid end points.

cookel2 commented:
I have now also tested the qid end points by running the Census Field Service with the fake case service. The result is successful in that I can see data at the following URLs: http://localhost:8161/cases/3305e937-6fb1-4ce1-9d4c-077f147789ac?caseEvents=false and http://localhost:8161/cases/3305e937-6fb1-4ce1-9d4c-077f147789ab?caseEvents=false

andrewjohnysons committed:
Added data to application.yml
Written config class to load the data and create maps
RestController uses maps to fulfill data responses.
Added new Fields to DTOs and deleted local versions
Split data into case.yml and questionnaires.yml
Split Config classes up.

andrewjohnysons committed:
Added data to application.yml
Written config class to load the data and create maps
RestController uses maps to fulfill data responses.
Added new Fields to DTOs and deleted local versions
Split data into case.yml and questionnaires.yml
Split Config classes up.
code review fixes
